compiler/cpp/src/generate/t_rs_generator.cc

```diff
@@ -290,13 +231,18 @@ void t_rs_generator::generate_typedef(t_typedef* ttypedef) {
   f_mod_ << "\n";
 }
 
+// Generates an enum, translating a thrift enum into a rust enum.
 void t_rs_generator::generate_enum(t_enum* tenum) {
   string ename = pascalcase(tenum->get_name());
-  indent(f_mod_) << "#[allow(dead_code)]\n";
-  indent(f_mod_) << "#[derive(PartialEq,Eq,Hash,Copy,Clone,Debug)]\n";
-  indent(f_mod_) << "pub enum " << ename << " {\n";
+  indent(f_mod_) << "enom! {\n";
```

Comment:
I have the same feeling re strukt and enom - maybe longer but grammatically-proper names will make more sense to me.